api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricHandler.java

```diff
@@ -847,7 +75,7 @@ public void onFailure(Throwable t) {
     public void findMetrics(
             @Suspended final AsyncResponse asyncResponse,
             @PathParam("tenantId") final String tenantId,
-        @ApiParam(value = "Queried metric type", required = true, allowableValues = "[num, avail, log]")
+        @ApiParam(value = "Queried metric type", required = true, allowableValues = "[gauge, avail, log]")
```

Comment:
Is there a reason why this is QueryParam instead of PathParam? It would make more sense as /{tenantId}/gauges or the proposed /{tenantId}/metrics and the type would only filter (by default send everything) ?